zstacklib/zstacklib/utils/fileUploader.py

```
@@ -0,0 +1,289 @@
+import hashlib
+
+import cherrypy
+import rbd
+from cherrypy._cpreqbody import Entity, Part, SizedReader
+from cherrypy._cprequest import Request
+
+import zstacklib.utils.http as http
+import zstacklib.utils.jsonobject as jsonobject
+import zstacklib.utils.plugin as plugin
+from zstacklib.utils.bash import *
```

Ruff (0.13.1)

`from zstacklib.utils.bash import *` used; unable to detect undefined names

(F403)